radix-engine-interface/src/api/types/indexed_value.rs

```diff
@@ -14,7 +14,7 @@ use utils::ContextualDisplay;
 #[derive(Clone, PartialEq, Eq)]
 pub struct IndexedScryptoValue {
     bytes: Vec<u8>,
-    references: HashSet<RENodeId>,
```

Comment:
Can we remove this change to `BTreeSet` for now since it isn't relevant to PR. From what I recall, Yulong changed these to `HashSet` due to really bad performance of `BTreeSet`, or it might have been somewhere else. Regardless, let's revert this for now.

Reply:
Sure - can remove it from this PR if you want.